### PR TITLE
Rebranding for HackUPC Fall 2016

### PR DESCRIPTION
**Pull request by casassg**
Includes:
- Change in colors
- Replace life.js to new animated background: leaves.js
- Corrections in FAQ
  And basically a full new edition of HackUPC! Ready to ship!

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
         </div>
         <div class="row top-description">
             <div class="col-md-12">
-                <p>Dream it. Hack it. Ship it.<br>500 hackers. 36h.<br>Barcelona, Spain | 7 - 9 Oct</p>
+                <p>Dream it. Hack it. Ship it.<br>500 hackers. 36h.</p>
```
Comment (rgascons) [inline dist/index.html:38]: Why do you remove the date?

Comment (casassg) [reply dist/index.html:38]: It didn't look good, and we didn't have it last season

File: dist/index.html [hunk 2]
```diff
@@ -139,5 +139,6 @@ <h1 class="partners-title">Partners</h1>
       ga('send', 'pageview');
 
     </script>
+    <span style="display: none;">BIENE HACKUPC!</span>
```
Comment (dasilvacontin) [inline dist/index.html:140]: 🚴

**Comment by dasilvacontin** [comment]
Good job. Gonna do some design tweaks and merge it.

**Comment by dasilvacontin** [comment]
@zakum42 @casassg I need collab access to the fork.

File: dist/index.html (hunk 1 above)
Comment (dasilvacontin) [reply dist/index.html:38]: The date is already on the header/title, so no need to repeat info on the same screen.

**Comment by rgascons** [comment]
@dasilvacontin Just sent you and inv

File: dist/index.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
 </head>
 
 <body>
-    <canvas id="lolcanvas" width="150" height="150" style="position:absolute;z-index:-100;top:0;left:0;width:100%;height:100%"></canvas>
+    <canvas id="lolcanvas" width="150" height="150" style="position:fixed;z-index:-100;top:0;left:0;width:100%;height:100%"></canvas>
```
Comment (casassg) [inline dist/index.html:20]: Not sure about this. It makes worse the FAQ legibility

File: src/styles/main.scss [hunk 4]
```diff
@@ -102,7 +106,7 @@ h1 {
     a {
         color: $blink;
         text-decoration: none;
-        font-family: 'Droid Sans', sans-serif;
+        font-family: minecraftia;
```
Comment (casassg) [inline src/styles/main.scss]: Readibility is not the best. We got a lot of complains for this last edition

Comment (dasilvacontin) [reply src/styles/main.scss]: Solution is not to use two completely different fonts though.

Comment (dasilvacontin) [reply src/styles/main.scss]: Actually, minecraftia was a rushed workaround for last edition – it was never meant to stay.

**Comment by dasilvacontin** [comment]
Not super happy, but can't afford sinking more time into this. Browser rendering inconsistencies help too.

screencapture-192-168-0-104-8081-dist-1472223822781

**Pull request merged**